java/client/src/org/openqa/selenium/remote/RemoteWebDriver.java

```diff
@@ -88,7 +91,8 @@
 public class RemoteWebDriver implements WebDriver, JavascriptExecutor,
       FindsById, FindsByClassName, FindsByLinkText, FindsByName,
       FindsByCssSelector, FindsByTagName, FindsByXPath,
-      HasInputDevices, HasCapabilities, Interactive, TakesScreenshot {
+      HasInputDevices, HasCapabilities, Interactive, TakesScreenshot,
+      HasVirtualAuthenticator {
```

Comment:
If this is only available in Chrome, it's probably best to add the marker interface and implementation to the `ChromiumDriver`. We can move the methods once this is in, unless you're particularly eager to make the change now.

Reply:
That's fine. Thank you for linking to the bug :)